docs/en/sql-reference/aggregate-functions/reference/exponentialmovingaverage.md

```diff
@@ -96,56 +96,56 @@ Result:
-│     0 │    3 │                0.054 │ ██▋                                       │
+│     0 │    2 │                0.058 │ ██▊                                        │
+│     0 │    3 │                0.054 │ ██▋                                        │
 │     0 │    4 │                0.051 │ ██▌                                        │
-│     0 │    5 │                0.047 │ ██▎                                       │
-│     0 │    6 │                0.044 │ ██▏                                       │
+│     0 │    5 │                0.047 │ ██▎                                        │
+│     0 │    6 │                0.044 │ ██▏                                        │
 │     0 │    7 │                0.041 │ ██                                         │
-│     0 │    8 │                0.038 │ █▊                                        │
-│     0 │    9 │                0.036 │ █▋                                        │
-│     0 │   10 │                0.033 │ █▋                                        │
+│     0 │    8 │                0.038 │ █▊                                         │
+│     0 │    9 │                0.036 │ █▋                                         │
+│     0 │   10 │                0.033 │ █▋                                         │
 │     0 │   11 │                0.031 │ █▌                                         │
-│     0 │   12 │                0.029 │ █▍                                        │
-│     0 │   13 │                0.027 │ █▎                                        │
-│     0 │   14 │                0.025 │ █▎                                        │
-│     0 │   15 │                0.024 │ █▏                                        │
+│     0 │   12 │                0.029 │ █▍                                         │
+│     0 │   13 │                0.027 │ █▎                                         │
+│     0 │   14 │                0.025 │ █▎                                         │
+│     0 │   15 │                0.024 │ █▏                                         │
 │     0 │   16 │                0.022 │ █                                          │
 │     0 │   17 │                0.021 │ █                                          │
-│     0 │   18 │                0.019 │ ▊                                         │
-│     0 │   19 │                0.018 │ ▊                                         │
-│     0 │   20 │                0.017 │ ▋                                         │
-│     0 │   21 │                0.016 │ ▋                                         │
-│     0 │   22 │                0.015 │ ▋                                         │
-│     0 │   23 │                0.014 │ ▋                                         │
-│     0 │   24 │                0.013 │ ▋                                         │
-│     1 │   25 │                0.079 │ ███▊                                      │
+│     0 │   18 │                0.019 │ ▊                                          │
```

Comment:
Could you please change the "slug" in l. 2 to camelCase? (it is all lowercase right now)